www/app/Services/Providers/CodexProvider.php

```
@@ -23,6 +23,18 @@ class CodexProvider extends AbstractCliProvider
      */
     private const PROJECT_DOC_MAX_BYTES = 500000; // ~488KB
 
+    /**
+     * Unique per-run system prompt filename to avoid overwriting user files.
+     * Generated in buildCliCommand(), used by write/cleanup methods.
+     */
+    private ?string $systemPromptFile = null;
```

PHPMD (2.15.0)

[warning] Avoid unused private fields such as '$systemPromptFile'. (undefined)

(UnusedPrivateField)